src/main.rs

```diff
@@ -215,6 +215,13 @@ fn camera_main(
         connected = true;
         info!("{}: Connected and logged in", camera_config.name);
 
+        let version_info = camera.version();
+        if version_info.is_ok() {
+            info!("{}: Camera reports firmware version {}", camera_config.name, version_info.unwrap().firmwareVersion);
+        } else {
+            info!("{}: Could not fetch version information", camera_config.name);
+        }
+
```

Comment:
This will report the version multiple times. One for the main stream and one for the substream. Could you move it into part of the `do_camera_managment` function that is the part of the code which only runs once per camera.

Reply:
Whoops, will do! I only have B400s here, maybe I should pick up a model that can do both streams as well.